assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -121,6 +122,10 @@ function MediaPane(props) {
     insertElement: state.actions.insertElement,
   }));
 
+  const [showRestrictAccessDialog, setShowRestrictAcccessDialog] = useState(
```

Comment:
That's a lot of c's! Maybe too many?

```suggestion
  const [showRestrictAccessDialog, setShowRestrictAccessDialog] = useState(
```

Secondarily, I prefer the naming style of an `is` or `has` prefix for booleans, but I'm not that rigid on it (anymore), so this is probably fine.

Reply:
Accepted, done!